pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/VedleggOpplysningerOmEtteroppgjoeret.kt

```diff
@@ -239,6 +216,7 @@ data class DuHarFaattAvviksBeloep(
 data class OmBeregningAvBarnetillegg(
     val barnetillegg: Expression<OpplysningerOmEtteroppgjoeretDto.Barnetillegg>,
     val periode: Expression<Year>,
+    val pensjonsgivendeInntekt: Expression<OpplysningerOmEtteroppgjoeretDto.InntektOgFratrekk>,
```

Comment:
Hvorfor tar du inn `pensjonsgivendeInntekt` her? Du bruker det for å bestemme om det skal stå tabell i entall eller flertall i frasen, men er det riktig når tabellene som vises i denne frasen er personinntekt?

Reply:
Jeg har tatt pensjonsgivendeInntekt inn der fordi jeg trenger styring av takster på linjer 432-457:
paragraph {
                showIf(
                    pensjonsgivendeInntekt.inntekt.inntekter.isNotEmpty()
                            and (pensjonsgivendeInntekt.fratrekk.fratrekk.isNotEmpty() or harFellesTillegg)
                ) {
                    textExpr(
                        Bokmal to "Tabellene under viser inntektene du".expr()
                                + ifElse(harFellesTillegg, " og annen forelder", "") + "",
                        Nynorsk to "Tabellene under viser inntektene du".expr()
                                + ifElse(harFellesTillegg, " og anna forelder", "") + "",
                        English to "The tables below show the personal incomes you".expr()
                                + ifElse(harFellesTillegg, " and the other parent", "") + ""
                    )
                }.orShow {
                    text(
                        Bokmal to "Tabellen under viser inntektene du",
                        Nynorsk to "Tabellen under viser inntektene du",
                        English to "The table below shows the personal incomes you"
                    )
                }
                textExpr(
                    Bokmal to " har hatt i ".expr() + periode.format() + ". Det er disse inntektene vi har brukt for å beregne barnetillegget.",
                    Nynorsk to " har hatt i ".expr() + periode.format() + ". Det er desse inntektene vi har brukt for å rekne ut barnetillegget.",
                    English to " have in ".expr() + periode.format() + ". These incomes were used to calculate your child supplement."
                )
            }

Reply:
Ja, jeg vet, men jeg mener du bruker feil data til det. Tabellene som faktisk skrives ut her er fra personinntekt som ligger under barnetillegg.

Reply:
Så jeg mener du burde bruke `barnetillegg.personinntekt.inntekt.inntekter.isNotEmpty()`.

Reply:
Tilfellet er vel faktisk at tabellen for `barnetillegg.personinntekt.inntekt` vises uansett, så betingelsen for om det skal stå `Tabellen` eller `Tabellene` burde være:
`barnetillegg.personinntekt.fratrekk.isNotEmpty() or harFellesTillegg`

Reply:
Jeg har rettet til val erflereTabellerPersonInntekt = (barnetillegg.personinntekt.fratrekk.fratrekk.isNotEmpty() or harFellesTillegg)